halo2-base/src/gates/range.rs

```diff
@@ -265,7 +328,11 @@ pub trait RangeInstructions<F: ScalarField> {
     }
 
     /// Returns `(c, r)` such that `a = b * c + r`.
-    ///
+    /// * a: value to divide
+    /// * b: value to divide by
+    /// * a_num_bits: number of bits in `a`
```

Comment:
Same as above, instead of number of bits of variable `a` (resp `b`), we are assuming `a` has at most `a_num_bits` bits